### DISCUSSION TITLE
freebsd

### DISCUSSION THREAD

**Discussion by Chetan-Saini1224** [tarantool/tarantool#5961] [Q&A] [answered]
can anyone please update the readme.freebsd .

**Comment by Totktonada** [comment, marked as answer]
We'll do in https://github.com/tarantool/tarantool-qa/issues/95 (at least we'll update testing dependencies).

But if you have some inputs, share them.

Update: The `README.FreeBSD` file was actualized in https://github.com/tarantool/tarantool/pull/6639.
